v4/feature/receipt/src/main/java/exchange/dydx/trading/feature/receipt/components/exchangerate/DydxReceiptExchangeRateViewModel.kt

```diff
@@ -42,9 +42,9 @@ class DydxReceiptExchangeRateViewModel @Inject constructor(
         transferInput: TransferInput?
     ): String? {
         val exchangeRate = transferInput?.summary?.exchangeRate ?: return null
-        val type = transferInput?.type ?: return null
-        val token = transferInput?.token ?: return null
-        val symbol = transferInput?.resources?.tokenResources?.toMap()?.get(token)?.symbol ?: return null
+        val type = transferInput.type ?: return null
+        val token = transferInput.token ?: return null
+        val symbol = transferInput.resources?.tokenResources?.toMap()?.get(token)?.symbol ?: return null
```

Comment:
nit: `.toMap()` is unnecessary, IMap should have all the same methods

Reply:
Updated